File: WordApp/Models/Sentences/SentenceModel.cs
using System.Collections.Generic;
using WordApp.Models.Base;

namespace WordApp.Models.Sentences
{
    public class SentenceModel: BaseModel
    {
        public string Text { get; set; }
        public List<SentenceTranslationModel> Translations { get; set; }
    }
}

File: WordApp/ClientApp/src/app/models/sentences/sentence.model.ts
import { SentenceTranslationModel } from './sentence-translation.model';

export class SentenceModel {
  text: string;
  translations: Array<SentenceTranslationModel>;
  id: string;
  rowVersion: string;

  constructor(text: string, translations: Array<SentenceTranslationModel>, id: string, rowVersion: string) {
    this.text = text;
    this.translations = translations;
    this.id = id;
    this.rowVersion = rowVersion;
  }
}
